### PR TITLE
Style overrides for YouTube Regrets page & fix nav

### LINKED ISSUE #3718: YouTube links and share buttons
Follow up from https://github.com/mozilla/foundation.mozilla.org/pull/3715#issue-324001057

- Can all text links be the standard #EA0B12 but with an underline
- Hover/Active/Select state be a lighter shard of #FD6976] with an underline

image

- Ideally the icon color would reverse on hover but please let me know what is possible (I'm not sure I understand the constraints)
image

### LINKED ISSUE #3723: YouTube Nav
Looks like the nav is missing the hamburger. It should have the it, like the other zen mode pages.

Screen cap from prod:
image

cc @mmmavis @Pomax

### PR DESCRIPTION
**Pull request by mmmavis**
Closes #3718 Style overrides for YouTube Regrets page
Closes #3723 Nav bug

Will assign code review after this passes design review

https://foundation-mofostaging-pr-3724.herokuapp.com/en/campaigns/youtube-regrets/

### CODE REVIEW

**Comment by kristinashu** [comment]
Perfect! But please capitalize the `L` in `Load more stories`

**Comment by kristinashu** [comment]
Nav looks good too!

File: network-api/networkapi/wagtailpages/models.py [hunk 1]
```diff
@@ -1435,4 +1435,10 @@ class YoutubeRegretsPage(FoundationMetadataPageMixin, Page):
         StreamFieldPanel('regret_stories'),
     ]
 
+    zen_nav = True
```
Comment (Pomax) [inline network-api/networkapi/wagtailpages/models.py]: The youtube page really should not need to care about this flag, its HTML etc. is bespoke. If we want the styling associated with the zen nav, then we should hardcode that into the template, not add a model migration.

**Comment by Pomax** [comment]
looks good to me

**Comment by mmmavis** [comment]
🎉

**Pull request merged**